### ISSUE TITLE
加载预训练权重后在自己数据集上训练初始准确率为0的问题

### ISSUE THREAD

**Issue by MEengle** [Topdu/OpenOCR#135]
### **当使用  pretrained_model:的时候，开始训练的时候Acc为0.**
Global:
  device: gpu
  epoch_num: 100
  log_smooth_window: 20
  print_batch_step: 10
  output_dir: ./output/svtrv2_smtr_gtc_rctc_ch
  save_epoch_step: [5, 1]
  eval_batch_step: [0, 100]
  eval_epoch_step: [0, 1]
  cal_metric_during_train: False
  pretrained_model: /root/projects/OpenOCR-main/pretrained_models/svtrv2_smtr_gtc_ch/best.pth
  checkpoints:
  use_tensorboard: false
  infer_img:

  character_dict_path: &character_dict_path ./tools/utils/ppocr_keys_v1.txt 
  max_text_length: &max_text_length 25
  use_space_char: &use_space_char False
  save_res_path: ./output/rec/ch/predicts_svtrv2_smtr_gtc_rctc_ch.txt
  use_amp: false

Optimizer:
  name: Adam
  lr: 0.0001 
  weight_decay: 0.0005
  filter_bias_and_bn: True

LRScheduler:
  name: CosineAnnealingLR
  warmup_epoch: 2

Architecture:
  model_type: rec
  algorithm: SVTRv2
  in_channels: 3
  Transform:
  Encoder:
    name: SVTRv2LNConvTwo33
    use_pos_embed: False
    dims: [128, 256, 384]
    depths: [6, 6, 6]
    num_heads: [4, 8, 12]
    mixer: [['Conv','Conv','Conv','Conv','Conv','Conv'],['Conv','Conv','FGlobal','Global','Global','Global'],['Global','Global','Global','Global','Global','Global']]
    local_k: [[5, 5], [5, 5], [-1, -1]]
    sub_k: [[1, 1], [2, 1], [-1, -1]]
    last_stage: True
    feat2d: false
  Decoder:
    name: CTCDecoder

Loss:
  name: CTCLoss
  zero_infinity: True

PostProcess:
  name: CTCLabelDecode
  character_dict_path: *character_dict_path
  use_space_char: *use_space_char

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: RatioDataSetTVResize
    ds_width: True
    padding: false
    data_dir_list: 
      - /opt/data/private/data/HW_Chinese/train_lmdb
    transforms:
      - DecodeImagePIL:
          img_mode: RGB
      - PARSeqAugPIL:
      - CTCLabelEncode: 
          character_dict_path: *character_dict_path
          use_space_char: *use_space_char
          max_text_length: *max_text_length
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] 
  sampler:
    name: RatioSampler
    scales: [[128, 32]]

    first_bs: &bs 64
    fix_bs: false
    divided_factor: [4, 16]
    is_training: True
  loader:
    shuffle: True
    batch_size_per_card: *bs
    drop_last: True
    max_ratio: &max_ratio 8
    num_workers: 4

Eval:
  dataset:
    name: RatioDataSetTVResize
    ds_width: True
    padding: False
    data_dir_list: 
      - /opt/data/private/data/HW_Chinese/test_lmdb
    transforms:
      - DecodeImagePIL: 
          img_mode: RGB
      - CTCLabelEncode: 
          character_dict_path: *character_dict_path
          use_space_char: *use_space_char
          max_text_length: *max_text_length
      - KeepKeys:
          keep_keys: ['image', 'label', 'length']
  sampler:
    name: RatioSampler
    scales: [[128, 32]]
  
    first_bs: *bs
    fix_bs: false
    divided_factor: [4, 16]
    is_training: False
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: *bs
    max_ratio: *max_ratio
    num_workers: 4

**### 但是不加载预训练权重的时候，初始Acc不为0。**
Global:
  device: gpu
  epoch_num: 100
  log_smooth_window: 20
  print_batch_step: 10
  output_dir: ./output/svtrv2_ctc/
  save_epoch_step: [15, 1]

  eval_epoch_step: [0, 1]
  eval_batch_step: [0, 100]
  cal_metric_during_train: True
  pretrained_model:
  checkpoints:
  use_tensorboard: false
  infer_img:

  character_dict_path: &character_dict_path ./tools/utils/ppocr_keys_v1.txt
  max_text_length: &max_text_length 25
  use_space_char: &use_space_char False
  save_res_path: ./output/rec/u14m_filter/predicts_svtrv2_ctc.txt
  use_amp: false 

Optimizer:
  name: AdamW
  lr: 0.0001 
  weight_decay: 0.0005
  filter_bias_and_bn: True

LRScheduler:
  name: OneCycleLR
  warmup_epoch: 1.5 
  cycle_momentum: False

Architecture:
  model_type: rec
  algorithm: SVTRv2
  Transform:
  Encoder:
    name: SVTRv2LNConvTwo33
    use_pos_embed: False
    out_channels: 256
    dims: [128, 256, 384]
    depths: [6, 6, 6]
    num_heads: [4, 8, 12]
    mixer: [['Conv','Conv','Conv','Conv','Conv','Conv'],['Conv','Conv','FGlobal','Global','Global','Global'],['Global','Global','Global','Global','Global','Global']]
    local_k: [[5, 5], [5, 5], [-1, -1]]
    sub_k: [[1, 1], [2, 1], [-1, -1]]
    last_stage: True
    feat2d: False
  Decoder:
    name: CTCDecoder

Loss:
  name: CTCLoss
  zero_infinity: True

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc
  is_filter: True

Train:
  dataset:
    name: RatioDataSetTVResize
    ds_width: True
    padding: False
    data_dir_list: 
      - /opt/data/private/data/HW_Chinese/train_lmdb
    transforms:
      - DecodeImagePIL:
          img_mode: RGB
      - PARSeqAugPIL:
      - CTCLabelEncode: 
          character_dict_path: *character_dict_path
          use_space_char: *use_space_char
          max_text_length: *max_text_length
      - KeepKeys:
          keep_keys: ['image', 'label', 'length']
  sampler:
    name: RatioSampler
    scales: [[128, 32]]
 
    first_bs: &bs 128
    fix_bs: false
    divided_factor: [4, 16] 
    is_training: True
  loader:
    shuffle: True
    batch_size_per_card: *bs
    drop_last: True
    max_ratio: 4
    num_workers: 4

Eval:
  dataset:
    name: RatioDataSetTVResize
    ds_width: True
    padding: False
    data_dir_list: 
    transforms:
      - DecodeImagePIL: 
          img_mode: RGB
      - CTCLabelEncode: 
          character_dict_path: *character_dict_path
          use_space_char: *use_space_char
          max_text_length: *max_text_length
      - KeepKeys:
          keep_keys: ['image', 'label', 'length']
  sampler:
    name: RatioSampler
    scales: [[128, 32]] 
   
    first_bs: 128
    fix_bs: false
    divided_factor: [4, 16] 
    is_training: False
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: *bs
    max_ratio: 4
    num_workers: 4
[2025/07/03 17:08:17] openrec INFO: epoch: [1/100], global_step: 10, lr: 0.000004, acc: 0.835937, norm_edit_dis: 0.835938, num_samples: 128.000000, loss: 256.106079, avg_reader_cost: 0.13599 s, avg_batch_cost: 3.20350 s, avg_samples: 128.0, ips: 39.95625 samples/s, eta: 6 days, 22:45:04
[2025/07/03 17:08:22] openrec INFO: epoch: [1/100], global_step: 20, lr: 0.000004, acc: 0.835937, norm_edit_dis: 0.835938, num_samples: 128.000000, loss: 253.700684, avg_reader_cost: 0.00581 s, avg_batch_cost: 0.44775 s, avg_samples: 128.0, ips: 285.87232 samples/s, eta: 3 days, 23:01:26
[2025/07/03 17:08:26] openrec INFO: epoch: [1/100], global_step: 30, lr: 0.000004, acc: 0.855469, norm_edit_dis: 0.855469, num_samples: 128.000000, loss: 248.671280, avg_reader_cost: 0.00571 s, avg_batch_cost: 0.45722 s, avg_samples: 128.0, ips: 279.95261 samples/s, eta: 2 days, 23:16:41
[2025/07/03 17:08:31] openrec INFO: epoch: [1/100], global_step: 40, lr: 0.000004, acc: 0.882812, norm_edit_dis: 0.882813, num_samples: 128.000000, loss: 241.821106, avg_reader_cost: 0.00540 s, avg_batch_cost: 0.45076 s, avg_samples: 128.0, ips: 283.96469 samples/s, eta: 2 days, 11:19:14
[2025/07/03 17:08:35] openrec INFO: epoch: [1/100], global_step: 50, lr: 0.000004, acc: 0.898437, no

**Comment by Topdu** [comment]
Training with this `./configs/rec/svtrv2/svtrv2_smtr_gtc_rctc_ch.yml` config for Chinese.
